.travis.yml

```diff
@@ -5,6 +5,7 @@ sudo: false
 cache:
   directories:
     - $HOME/.composer/cache
+    - $HOME/.composer/vendor
```

Comment:
What's the difference between `$HOME/.composer/cache` and `$HOME/.composer/vendor`?